### PR TITLE
Wire-up Content

### PR DESCRIPTION
**Pull request by mrinalwadhwa**

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@sobi3ch one thing to note in the shell script is that I removed the github token ... our deps are all public.

**Comment by sobi3ch** [comment]
> @sobi3ch one thing to note in the shell script is that I removed the github token ... our deps are all public.

Yesterday I want to write to you about this but release just that :)

File: .gitignore [hunk 1]
```diff
@@ -4,8 +4,11 @@ node_modules
 *DS_Store
 *.env
 .idea/
-ockam-sample-documentation
 .gitmodules
+
+src/content/learn/documentation
+src/content/learn/proposals
+
```
Comment (sobi3ch) [inline .gitignore]: Don't you think sub-folder `src/content/` should be ignored all together except those directories we already have? 
```
blog/
samples/
```

We could achieve this like this
E.g.:
```
!src/content/
src/content/*
!src/content/blog/
!src/content/samples/
```
https://stackoverflow.com/a/5534865/473390

Then we don't need to remember about adding new dirs we don't want to track but only those we want to.

Comment (mrinalwadhwa) [reply .gitignore]: makes sense, I'll change.

**Pull request merged**